### DISCUSSION TITLE
Is this just for Java?

### DISCUSSION THREAD

**Discussion by huyz** [Vanco/SequencePlugin#105] [Q&A]
Confusingly, there's no mention of programming language anywhere in the plugin description or Github.  I assume it's for Java, but IntelliJ supports many more languages other than Java.
Could you please clarify?

**Comment by Vanco** [comment]
Yes, It is for JAVA! And Kotlin (Partial).

**Comment by Vanco** [comment]
from v3.x, support Java, Kotlin, Scala(Beta) and Grovvy(limited).
